docs/extensions/tutorials/_posts/1970-01-02-WritingATheme.md

```diff
@@ -281,17 +329,18 @@ Open `Style` tab and choose `Customize theme`. Theme variables are grouped into
     // other variables
   },
   "layout": {
-    "sections": [{
+    "sections": [
+    {
```

Comment:
It's good that you're detailed and you check all these things :). Here, the bug is in the library that CLI is using which formats the JSON in that way. The correct way is to format it in the way that was already written.